### PR TITLE
AB#5020

### PR DESCRIPTION
**Pull request by sachin755**
### Proposed changes

*
*

### Related issues

*
*

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by johnjdl** [comment]
Tested with the local build, @DallasP9124 @r-joyce please review and merge.

File: opencti-platform/opencti-front/src/private/components/riskAssessment/risks/remediations/RemediationDetailsPopover.js [hunk 1]
```diff
@@ -102,23 +117,51 @@ class RemediationDetailsPopover extends Component {
...
+        input: finalValues,
+      },
+      setSubmitting,
+      onCompleted: (data) => {
+        setSubmitting(false);
+        resetForm();
+        this.handleClose();
+        this.props.history.push(`/activities/risk assessment/risks/${this.props.riskId}/remediation/`);
+      },
+      onError: (err) => console.log('editRemediationMutationError', err),
```
Comment (BigBallard) [inline opencti-platform/opencti-front/src/private/components/riskAssessment/risks/remediations/RemediationDetailsPopover.js]: @sachin755 do not use `console.log` for errors. Use `console.error` and expose an error message to the user using `utils/bakesToast.js`.

**Review by BigBallard — CHANGES_REQUESTED** [review]

**Review by BigBallard — APPROVED** [review]

**Comment by BigBallard** [comment]
@r-joyce good, thank you @sachin755 @AMAN-SHARMA-82591

**Review by r-joyce — APPROVED** [review]
Looks good to me 👍

**Pull request merged**